server/routers/growerAccountsRouter.ts

```diff
@@ -20,7 +20,7 @@ router.get(
         limit: Joi.number().integer().min(1).max(1000),
         offset: Joi.number().integer().min(0),
         keyword: Joi.string(),
-        organization_id: Joi.string().uuid(),
+        organization_id: Joi.array(),
```

Comment:
We can use Joi to add an extra level of validations on each item in the array to ensure they are uuids

Joi.array.items(Joi.string.uuid())